Composer/packages/client/src/components/DialogWrapper/index.tsx

```diff
@@ -1,19 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import React from 'react';
-import { Dialog, DialogType, IDialogProps } from 'office-ui-fabric-react/lib/Dialog';
+import React, { useEffect, useState } from 'react';
+import { Dialog, DialogType, IDialogProps, IDialogContentStyles } from 'office-ui-fabric-react/lib/Dialog';
+import { IModalStyles } from 'office-ui-fabric-react/lib/Modal';
 
 import { styles } from './styles';
 
 interface DialogWrapperProps extends Pick<IDialogProps, 'onDismiss'> {
   isOpen: boolean;
   title: string;
   subText: string;
+  overrideStyles?: { modal: Partial<IModalStyles>; dialog: Partial<IDialogContentStyles> };
 }
 
 export const DialogWrapper: React.FC<DialogWrapperProps> = props => {
-  const { isOpen, onDismiss, title, subText, children } = props;
+  const { isOpen, onDismiss, title, subText, children, overrideStyles } = props;
+  const [wrapperStyle, updateWrapperStyle] = useState({
+    modal: styles.modal,
+    dialog: styles.dialog,
+  });
+
+  useEffect(() => {
+    if (overrideStyles) {
+      updateWrapperStyle({
+        modal: overrideStyles.modal,
+        dialog: overrideStyles.dialog,
+      });
+    }
+  }, [overrideStyles]);
```

Comment:
Is this necessary? Can we just pass the prop directly to the components?

Reply:
Yeah.. I was thinking of doing it but there were a few other components that were using the DialogWrapper and all of them followed the same style which meant it was using the styles.ts from the DialogWrapper folder itself. I figured it's safer to not touch them now rather override the style through props. But I agree importing the style in the individual components and passing it through to DialogWrapper seems better